utils/chaosengine.j2

```diff
@@ -9,7 +9,6 @@ spec:
     runner:
       image: "litmuschaos/chaos-executor:ci"
       type: "go"   
-  jobCleanUpPolicy: retain
   monitoring: false
```

Comment:
is `jobCleanUpPolicy: retain` not required?

Reply:
You can set it to delete also

Reply:
yes!